DGWebApi/Utils/LocationUtils.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Globalization;
+
+namespace DGWebApi.Utils
+{
+    public class LocationUtils
+    {
+        public static string GetNameFromCountryCode(string code)
+        {
+            var cultureInfo = new CultureInfo(code);
+            var ri = new RegionInfo(cultureInfo.Name);
```

Comment:
As discussed, this throws an error when incorrect codes are used

Reply:
I've added a try catch block. Let me know if there is a better way to handle this please.